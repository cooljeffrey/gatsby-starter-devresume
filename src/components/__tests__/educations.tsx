import React from "react";
import renderer from "react-test-renderer";
import Educations from "../educations";
describe("Educations", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Educations
          items={[
            {
              degree: "Degree",
              institution: "Institution",
              dateStart: "2020",
              dateEnd: "2030",
            },
          ]}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
