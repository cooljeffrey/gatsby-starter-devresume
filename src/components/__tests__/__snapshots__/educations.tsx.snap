// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Educations renders correctly 1`] = `
<section
  className="education-section py-3"
>
  <h3
    className="text-uppercase resume-section-heading mb-4"
  >
    Education
  </h3>
  <ul
    className="list-unstyled resume-education-list"
  >
    <li
      className="mb-3"
    >
      <div
        className="resume-degree font-weight-bold"
      >
        Degree
      </div>
      <div
        className="resume-degree-org text-muted"
      >
        Institution
      </div>
      <div
        className="resume-degree-time text-muted"
      >
        2020
         - 
        2030
      </div>
    </li>
  </ul>
</section>
`;
